docs/installation_quickstart.rst

```diff
@@ -55,8 +55,12 @@ being picked up correctly and that the tracer will be able to connect to the Dat
 Note: ``--info`` Only reflects configurations made via environment variables, not those made in code.
 
 
-If ``ddtrace-run`` isn't suitable for your application then :py:func:`ddtrace.patch_all`
-can be used::
+When ``ddtrace-run`` cannot be used, a similar start-up behavior can be achieved
+with the import of ``ddtrace.auto``. This should normally be imported as the
+first thing during the application start-up.
+
```

Comment:
```suggestion
first thing during the application start-up::

    import ddtrace.auto
    # Rest of startup code...

```
Maybe something like this? I think this is your discretion if you think adding this snippet will be helpful. I personally don't think adding this would be useful since we are just adding a single line to the top of a startup script.